### ISSUE TITLE
with Amazon S3Sinks Not able to transform or control what's written to the s3bucket 

### ISSUE THREAD

**Issue by vsrinivasan08** [hazelcast/hazelcast#24856]
With Amazon **S3Sinks** how can we control what's sinking to the S3 file ? Let's Say I want to write only values to the s3 sinking files without keys from the map ? Also cannot able to control extension of the sinking file and content type too.

ref - https://docs.hazelcast.org/docs/5.0.2/javadoc/com/hazelcast/jet/s3/S3Sinks.html

```
const capitalcities = await hzClient.getMap('capitals');
await capitalcities.put( 1, "Tokyo" );
await capitalcities.put( 2, "Honkong" );
await capitalcities.put( 3, "Indonesia" );
```

Need only values from map - `capitals` to be written to s3. Not able to achieve this with existing transformation docs
```
Pipeline p = Pipeline.create();
 p.readFrom(Sources.map("capitals"))
  .writeTo(S3Sinks.s3("bucket", "my-map-", StandardCharsets.UTF_8,
      () -> S3Client.create(),
      Object::toString
 ));
```

**Current Behaviour in S3 file:**
```
1=Tokyo
2=Hongkong
3=Indonesia
```
**Expected Behaviour in S3 file:**
```
Tokyo
Hongkong
Indonesia
```
Basically I couldnot able to find any thing even in the transformation docs - https://docs.hazelcast.com/hazelcast/5.3/pipelines/transforms

Thanks
